webfonts/README.md

```diff
@@ -2,17 +2,69 @@
 
 ## Font file CSS
 
-All static fonts are available via `./red-hat-font.css`
+Original fonts are available via `./font.css`
 
-Variable fonts are available via `./red-hat-font-variable.css`
+Modified fonts are available via `./red-hat-font.css`
 
 Minified versions of both of these are available at:
 
+* `./font.min.css`
 * `./red-hat-font.min.css`
-* `./red-hat-font-variable.min.css`
 
 ## Local development
 
 Generate new minified versions of the CSS files by running `npm run minifiy` at the project root.
```

Comment:
This has an extra `i` 

should be `npm run minify`

Reply:
Yikes! Will fix, thanks!